app/frontend/src/features/BugReport.tsx

```diff
@@ -9,27 +9,33 @@ import {
   DialogTitle,
 } from "components/Dialog";
 import { BugIcon } from "components/Icons";
+import SuccessSnackbar from "components/SuccessSnackbar";
 import TextField from "components/TextField";
 import { useAuthContext } from "features/auth/AuthProvider";
 import {
-  BUG_DESCRIPTION,
-  EXPECT,
-  PROBLEM,
+  BUG_DESCRIPTION_HELPER,
+  BUG_DESCRIPTION_NAME,
+  BUG_REPORT_SUCCESS,
+  CANCEL,
+  EXPECT_HELPER,
+  EXPECT_NAME,
+  PROBLEM_HELPER,
+  PROBLEM_NAME,
   REPORT,
-  STEPS,
   SUBMIT,
   WARNING,
 } from "features/constants";
 import { Error as GrpcError } from "grpc-web";
+import { ReportBugRes } from "pb/bugs_pb";
 import React, { useState } from "react";
 import { useForm } from "react-hook-form";
 import { useMutation } from "react-query";
+import { Link } from "react-router-dom";
```

Comment:
Sorry just seen this... I think we want to use `Link` from Material UI instead like Lucas said so we get the right link styling from the theme. Also `Link` from `react-router-dom` is only meant for links between our sites and not to external URLs

Reply:
Yeah, thanks. My bad, I was told in the meeting to use it... but I was having trouble with it and this Link was used in other spots too, so I used it...

Fixed now

Reply:
Yeah it's confusing they are named the same thing, I'm sure you won't be the last person to be confused by this :p